source/disassemble.cpp

```diff
@@ -49,6 +49,8 @@ namespace {
 class Disassembler {
   enum { kStandardIndent = 15 };
 
+  using out_stream = libspirv::out_stream;
```

Comment:
using libspirv::clr didn't work.  The compiler wanted a type, but it's only a namespace containing functions.  I didn't look further.